docs/nydus-zran.md

```diff
@@ -39,6 +39,7 @@ Or you can generate one by `nydusify` tool:
 # Convert the existing OCI image `your-registry.com/node:19.0` to `your-registry.com/node:19.0-nydus-oci-ref`:
 sudo nydusify convert --oci-ref --source your-registry.com/node:19.0 --target your-registry.com/node:19.0-nydus-oci-ref
 ```
+> **Tip**: The nydus zran artifact must have the same namespace with the OCI image
```

Comment:
zran -> ZRAN?

Reply:
**Tips**: 
- Nydus ZRAN artifacts must be in the same namespace with the OCI image.